projects/observability/src/pages/custom-dashboards/custom-dashboard.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { RestClientService } from '@hypertrace/common';
+import { Observable } from 'rxjs';
+import { DashboardDefaultConfiguration } from './../../shared/dashboard/dashboard-wrapper/navigable-dashboard.module';
+
+export interface DashboardListItem {
+  link: string;
+  display: string;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CustomDashboardService {
+  public static readonly API_ID_PARAM_NAME: string = 'dashboard_id';
+
+  public constructor(private readonly restClient: RestClientService) {}
+  public fetchDashboards(): Observable<DashboardListItem[]> {
+    return this.restClient.Get<DashboardListItem[]>('/list.json');
+  }
+  public fetchDashboard(dashboardId: string): Observable<DashboardDefaultConfiguration> {
```

Comment:
There's already a `fetchDashboards` method in the same class. Naming might get confusing.

How about we rename this method to something more specific, like, `fetchDashboardConfigById()`?

Reply:
Noted